Explorer/Assets/Scripts/MVC/IWindowsStackManager.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace MVC
+{
+    public interface IWindowsStackManager

```

Comment:
Let's make this class internal - I have a feeling it should not be exposed

Reply:
If it's internal it can be used in the constructor of MVCManager